android/library/src/main/java/com/liferay/mobile/screens/ScreensInitProvider.java

```diff
@@ -9,13 +9,16 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import com.liferay.mobile.screens.context.LiferayScreensContext;
+import com.liferay.mobile.screens.thingscreenlet.delegates.ConverterDelegate;
 
 public class ScreensInitProvider extends ContentProvider {
 
 	@Override
 	public boolean onCreate() {
 		Context context = getContext().getApplicationContext();
 		LiferayScreensContext.init(context);
+
+		ConverterDelegate.initializeConverter();
```

Comment:
This could be done in another provider to separate responsabilities. But don't change it.